rpi-clone

```diff
@@ -573,7 +573,7 @@ get_src_disk()
 	partition=${1#/dev/}
 	disk=${partition:: -1}
 	num="${partition: -1}"
-	if [[ $disk == *"mmcblk"* ]]
+	if [[ $disk == *"mmcblk"* || $disk == *"nvme"* ]]
 	then
 		SD_slot_boot=1
 		disk=${disk:0:7}
```

Comment:
Just a comment: I first thought this line will fail for nvme but it works fine :+1: . It works for nvme only as long as for /dev/nvme<x>n<y> x and y has only one digit. But that's a restriction also on the previous code which extracts the partition number.